packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -15,6 +15,14 @@ const FeedbackStore = types
     onHide: () => true
   }))
   .views(self => ({
+    get applicableRules () {
```

Comment:
What are these rules applicable to? I thought it might be a specific task, but the code isn't filtering on task key.

Reply:
Maybe could use a better name, but essentially this is getting the rules that have messages (success or failure) that will be shown. The updated test rules show an example where I'd expect the `classification.metadata.feedback` to include three rules, one of which a message was never shown, but `applicableRules` is designed to return the 2 rules for which a message will be shown.